xbmc/ServiceManager.h

```diff
@@ -29,7 +29,9 @@ class CBinaryAddonCache;
 }
 
 namespace ActiveAE {
+class CActiveAE;
 class CActiveAEDSP;
+class CActiveAE;
```

Comment:
thx @garbear 
should be fixed with this [PR](https://github.com/FernetMenta/xbmc/pull/420/files#diff-e0852c06443e72e1a8e07ce903e1406a).